ts/src/bitso.ts

```diff
@@ -1722,7 +1714,7 @@ export default class bitso extends Exchange {
             'txid': this.safeString (details, 'tx_hash'),
             'timestamp': this.parse8601 (datetime),
             'datetime': datetime,
-            'network': this.safeNetwork (networkId),
+            'network': this.networkIdToCode (networkId),
```

Comment:
same thing here, this is a different behavior

Reply:
pushed